src/library/styles/components/_button.scss

```diff
@@ -63,15 +63,27 @@ a {
     }
   }
 
+  &.bsds-button-assertive {
+    color: $neutral-100;
+  }
+
+  &.bsds-button-subtle {
+    color: $action-primary-40;
+
+    &:hover {
+      @include button-hover;
+    }
+  }
+
   &.bsds-button,
   &.bsds-button-subtle {
     &:visited {
       color: $action-primary-40;
     }
   }
```

Comment:
i think we can condense these too since bsds-button and bsds-button-subtle also share the same text color